src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -200,6 +200,13 @@ elementFormDefault="qualified">
                   </xs:documentation>
                 </xs:annotation>
               </xs:element>
+              <xs:element name="NoWarn">
+                <xs:annotation>
+                  <xs:documentation>
+                    <!-- _locID_text="PackageReference_NoWarn" _locComment="" -->Comma-separated list of warning codes to ignore (such as NU1605)
```

Comment:
I haven't checked, but I think both commas and semicolons work? I'm assuming this is from some other docs.

Reply:
Nope that's my guess, and I'm by no means an expert. Edits are welcome, feel free to push to this PR directly.

Reply:
argh, I mean to write Semicolon-separated, will need to fix

Reply:
Just tested it, and semicolon-separated seems to be correct.

Reply:
Csc and Vbc accept both commas and semicolons. I believe we do too, but I'd rather encourage semicolons.

Reply:
We don't:
```XML
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net6.0</TargetFramework>
    <ImplicitUsings>enable</ImplicitUsings>
    <Nullable>enable</Nullable>
    <NoWarn>NAT011,NAT012</NoWarn>
  </PropertyGroup>

  <Target Name="AlwaysRun">
    <Warning Code="NAT011" Text="You fail" />
    <Warning Code="NAT012" Text="Other Fail" />
  </Target>

</Project>
```

```
C:\Users\forgind\Desktop\Archives\Bug-specific\myTemp>dotnet build myTemp.csproj /t:AlwaysRun
Microsoft (R) Build Engine version 17.0.0+c9eb9dd64 for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
  All projects are up-to-date for restore.
C:\Users\forgind\Desktop\Archives\Bug-specific\myTemp\myTemp.csproj(12,5): warning NAT011: You fail
C:\Users\forgind\Desktop\Archives\Bug-specific\myTemp\myTemp.csproj(13,5): warning NAT012: Other Fail

Build succeeded.

C:\Users\forgind\Desktop\Archives\Bug-specific\myTemp\myTemp.csproj(12,5): warning NAT011: You fail
C:\Users\forgind\Desktop\Archives\Bug-specific\myTemp\myTemp.csproj(13,5): warning NAT012: Other Fail
    2 Warning(s)
    0 Error(s)

Time Elapsed 00:00:00.69
```

Reply:
We should probably fix that now that we're promoting NoWarn to MSBuildWarningsAsMessages. https://github.com/dotnet/msbuild/issues/7094